builder/azure/arm/builder.go

```diff
@@ -111,31 +116,31 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 	if err := resolver.Resolve(&b.config); err != nil {
 		return nil, err
 	}
+
 	if b.config.ClientConfig.ObjectID == "" {
-		b.config.ClientConfig.ObjectID = getObjectIdFromToken(ui, spnCloud)
+		b.config.ClientConfig.ObjectID = *objectID
```

Comment:
Is there ever a case where objectID is nil and this throws a nil pointer deference error?